src/app/icd/client/ICDClientStorage.h

```diff
@@ -24,11 +24,17 @@
 #include <lib/core/DataModelTypes.h>
 #include <lib/core/ScopedNodeId.h>
 #include <lib/support/CodeUtils.h>
+#include <lib/support/CommonIterator.h>
+#include <protocols/secure_channel/CheckinMessage.h>
 #include <stddef.h>
 
 namespace chip {
 namespace app {
 
+// 4 bytes for counter + 2 bytes for ActiveModeThreshold
+inline constexpr uint8_t kAppDataLength = 6;
```

Comment:
This does not make sense.  Why is `chip::app::kAppDataLength` equal to 6?

This needs to be scoped to some more narrow namespace (e.g. be inside ICDClientStorage or some other ICD-related class).

Reply:
@mkardous-silabs Where do you recommend moving this?

Reply:
If i am not mistaken, it is only used in 1 class. Could be a static constexpr within that class instead of a global

Reply:
Okay, I'll move it to the class and make it static.